### ISSUE TITLE
[LATTE-dev] Cannot create vetting report in non-FFI mode

### ISSUE THREAD

**Issue by orionlee** [noraeisner/LATTE#19]
In the current `LATTE-dev` branch, the vetting report pdf cannot be generated, after all the analysis was completed.

```
Pyaneti has not been installed so you can't model anything yet. Contact Nora or Oscar for the LATTE version of the Pyaneti code.
Traceback (most recent call last):
  File "C:\pkg\_winNonPortables\Anaconda3\envs\LATTE-dev\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\pkg\_winNonPortables\Anaconda3\envs\LATTE-dev\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\dev\_juypter\LATTE\LATTE\__main__.py", line 546, in <module>
    utils.interact_LATTE(tic, indir, syspath, sectors_all, sectors, ra, dec, args)  # the argument of whether to shos the images or not
  File "C:\dev\_juypter\LATTE\LATTE\LATTEutils.py", line 690, in interact_LATTE
    brew.brew_LATTE(tic, indir, syspath, transit_list, simple, BLS, model, save, DV, sectors, sectors_all, alltime, allflux, allflux_err, all_md, alltimebinned, allfluxbinned, allx1, allx2, ally1, ally2, alltime12, allfbkg, start_sec, end_sec, in_sec, tessmag, teff, srad, ra, dec, args)
  File "C:\dev\_juypter\LATTE\LATTE\LATTEbrew.py", line 378, in brew_LATTE
    ldv.LATTE_DV(tic, indir, syspath, transit_list, sectors_all, target_ra, target_dec, tessmag, teff, srad, [0], [0], tpf_corrupt, astroquery_corrupt, FFI = False,  bls = False, model = model, mpi = args.mpi)
TypeError: LATTE_DV() missing 5 required positional arguments: 'c_id', 'bls_stats1', 'bls_stats2', 'tpf_corrupt', and 'astroquery_corrupt'
```

The latest commit https://github.com/noraeisner/LATTE/commit/5eea005668f67b1e98e554c557119508c8e92805 about report data broke the code. It worked after I backed out the commit.

**Comment by orionlee** [comment]
It looks like with commit 5eea005, the FFI path `brew_LATTE_FFI()` was updated to use the new `LATTE_DV` call, but the non-FFI path `brew_LATTE()` is not updated.

**Comment by orionlee** [comment]
Fixed in commit 20280dc999a74eee4bca0d9cf3ecd8bff40a6af6
